cmd/root.go

```diff
@@ -70,11 +68,11 @@ func init() {
 
 func initConfig() {
 	if configPath == defaultConfigPath {
-		val, present := os.LookupEnv(configFileEnvVar)
+		val, present := os.LookupEnv(config.ConfigFilePath)
 		if present {
 			configPath = val
 		} else {
-			val, present = os.LookupEnv(configFileEnvVarOld)
+			val, present = os.LookupEnv(config.ConfigFilePathOld)
 			if present {
 				configPath = val
 			}
```

Comment:
What do you think of also moving the contents of the `if` to `config/environment.go`?

Reply:
Sounds reasonable, i moved the environment files to a separate package as there may config unrelated variables. Will also move the loading logic to this package.